### PR TITLE
[VGNWB-323] Refactor Concept auth config

### PR DESCRIPTION
**Pull request by nowke**
* Get rid of `configuration` attribute for Concept authority models
* Add separate classes for `Conceptpower` and `VIAF` with their corresponding API calls as class methods
* Add test cases for `Conceptpower` and `VIAF` classes
* Remove `restable` and `conceptpower` packages - which are not used anymore after this refactor

Related Frontend PR - https://github.com/diging/vogon-web-client/pull/54

### CODE REVIEW

**Review by yuvakurakula — APPROVED** [review]

**Comment by tjquinn1** [comment]
I can't search for concepts. I get `AttributeError: 'NoneType' object has no attribute 'search'`. Is there some setup I need to add someplace for this to work? If there is lets document it in confluence.

**Comment by nowke** [comment]
[VGNWB-323] Documented the prerequisites in Confluence - https://diging.atlassian.net/wiki/spaces/VGNWB/pages/2086273044/Vogon+Initial+Setup

[VGNWB-323]: https://diging.atlassian.net/browse/VGNWB-323

**Comment by tjquinn1** [comment]
[VGNWB-323] This is a test

[VGNWB-323]: https://diging.atlassian.net/browse/VGNWB-323

**Comment by tjquinn1** [comment]
fix conflicts

**Comment by tjquinn1** [comment]
fix build conflicts

**Comment by nowke** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

File: goat/models.py [hunk 1]
```diff
@@ -28,17 +26,16 @@ class Authority(BasicAccessionMixin):
...
     def manager(self):
-        if not self.configuration:
-            raise AttributeError("Configuration unavailable for %s" % self.name)
-        return AuthorityManager(self.configuration)
+        if self.builtin_identity_system.name == 'builtin:Conceptpower':
+            from concepts.conceptpower import ConceptPower
+            return ConceptPower()
+        if self.builtin_identity_system.name == 'builtin:VIAF':
+            from concepts.viaf import Viaf
+            return Viaf()
```
Comment (tjquinn1) [inline goat/models.py:38]: If we add another identity system this will have to be updated right? If so is that documented?

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

File: goat/models.py (hunk 1 above)
Comment (nowke) [reply goat/models.py:38]: Yes. We need to write a whole class similar to one of these for adding another type of identity system. I updated it in the documentation https://diging.atlassian.net/wiki/spaces/VGNWB/pages/2086273044/Vogon+Initial+Setup

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by nowke** [comment]
Will reopen the ticket after fixing this - https://diging.atlassian.net/browse/VGNWB-222

**Comment by nowke** [comment]
Fixes VGNWB-222 (https://diging.atlassian.net/browse/VGNWB-222)

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Pull request merged**